### PR TITLE
Tiny lint fixes

### PR DESCRIPTION
**Pull request by simondeziel**
Address the feedback left in #10617 after it was merged and a few more things.

### CODE REVIEW

File: test/lint/negated-is-bool.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 echo "Checking usage of negated shared.Is(True|False)*() functions..."
 
-OUT=$(grep -Pr --exclude-dir=.git '!(shared\.)?Is(True|False).*\(' . 2>/dev/null || true)
+OUT=$(git grep -P '!(shared\.)?Is(True|False).*\(' '*.go' || true)
```
Comment (tomponline) [inline test/lint/negated-is-bool.sh:5]: Does git grep work also on uncommitted files? If not I think we should stop using it for these static analysis checks as it should work on uncommitted files when run locally.

File: test/lint/newline-after-block.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 echo "Checking that functional blocks are followed by newlines..."
 
-files=$(find ./ -type f -name "*.go")
+files=$(git ls-files "*.go")
```
Comment (tomponline) [inline test/lint/newline-after-block.sh:5]: AFAIK this doesn't include uncommitted files, if that is the case, then I don't think we want this change as running `make static-analysis` should work on uncommitted files.

**Review by tomponline — CHANGES_REQUESTED** [review]
I think we need to check and make sure that running `make static-analysis` works the same for uncommitted files as it does for committed files.

**Comment by tomponline** [comment]
@markylaing the tests workflow is always failing the unit tests on TestConvertNetworkConfig btw.

**Comment by markylaing** [comment]
> @markylaing the tests workflow is always failing the unit tests on TestConvertNetworkConfig btw.

@tomponline ah I see, GitHub actions stopped working when I added unit tests. I should have put it back to draft to make sure. Not sure why it is failing but we can remove the unit tests and reinstate unit tests for the client on Ubuntu in the meantime.

**Comment by tomponline** [comment]
> > @markylaing the tests workflow is always failing the unit tests on TestConvertNetworkConfig btw.
> 
> @tomponline ah I see, GitHub actions stopped working when I added unit tests. I should have put it back to draft to make sure. Not sure why it is failing but we can remove the unit tests and reinstate unit tests for the client on Ubuntu in the meantime.

Ok thanks that would be good. I'm sure I've seen that failure locally too in the past so hopefully easy to reproduce.

**Comment by markylaing** [comment]
@tomponline I've removed the unit tests in #10621 - it should pass static analysis now.

**Pull request merged**